widgets/alternate/alternate.html.twig

```diff
@@ -0,0 +1,22 @@
+<div class="vud-widget vud-widget-alternate" id="{{ id }}">
```

Comment:
Please see [indenting coding standards](https://www.drupal.org/docs/develop/standards/coding-standards#indenting), they apply to twig too.